packages/calcite-components/src/components/card/card.scss

```diff
@@ -3,18 +3,27 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ *
+ * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
  * @prop --calcite-card-background-color: Specifies the background color of the component.
  * @prop --calcite-card-border-color: Specifies the border color of the component.
- * @prop --calcite-card-shadow: Specifies the shadow of the component.
+ * @prop --calcite-card-checkbox-background-color-checked: *deprecated* defines the background-color of the sub-component when checked.
+ * @prop --calcite-card-checkbox-background-color: *deprecated* defines the background-color of the sub-component.
```

Comment:
Maybe a convention outside the scope of this, but can we use **[Deprecated]** for this tag?

Reply:
Maybe we can have a custom stylelint rule for this? Deprecated should be surrounded by brackets or something.

Reply:
Yeah, that's a good idea. Both `@prop` here and `@slot` in the `tsx` files use this as unfortunately you can't use the built-in JSDoc `@deprecated` functionality. Maybe something Stencil could build into their compiler.

Reply:
We could open a stencil request for this but i'm not sure if they would add it.

Reply:
Would it be the same as this issue @macandcheese ? https://github.com/ionic-team/stencil/issues/3111

Reply:
Similar - the JSDoc `@deprecated` they mention in the follow-up comment works for props, events, methods, just not for slots / css props.